main.scss

```diff
@@ -22,27 +24,55 @@ $_o-quote-first-import: true !default;
 ///     	'editorial': true
 ///     ));
 ///
-/// @param {Map} $opts [('cite': true, 'standard': true] - The feature set to output rules for.
+/// @param {Map} $opts [('cite': true, 'standard': true, 'editorial': true] - The feature set to output rules for.
 /// @output The output includes the `.o-quote*` selectors
 @mixin oQuote($opts: (
-	cite: true,
-	standard: true
+	'cite': true,
+	'standard': true,
+	'editorial': true,
 )) {
+	// Enabled if the option is given and the current brand supports the feature.
+	$standard-enabled: map-get($opts, 'standard') and _oQuoteSupports('standard');
+	$editorial-enabled: map-get($opts, 'editorial') and _oQuoteSupports('editorial');
 	$cite-enabled: map-get($opts, 'cite');
-	$standard-enabled: map-get($opts, 'standard');
 
+	// Output base quote styles.
 	.o-quote {
-		@include oQuoteBase();
+		@extend %_o-quote;
```

Comment:
Thinking out loud: I hardly see Origami styles using `@extend`.

Reply:
In the past we have strongly discouraged them. They can lead to an unpredictable source order and previously caused us problems particularly as placeholders would be output multiple times, whenever a component was imported. We avoid most issues by only outputting placeholders once with a flag and keeping them private behind mixins. o-quote is a bit of an experiment to see if there's anything else we haven't considered!